cheroot/server.py

```diff
@@ -39,6 +39,21 @@
                 if req.close_connection:
                     return
 
+For running a server you can invoke :func:`start() <HTTPServer.start()>` (it
```

Comment:
Have you tried rendering it? I think it's a broken reference.

Reply:
I rendered via

    pip install '.[docs]' && sphinx-build docs docs/_build

it did this HTML:

```
<p>For running a server you can invoke <a class="reference internal" href="#cheroot.server.HTTPServer.start" title="cheroot.server.HTTPServer.start"><code class="xref py py-func docutils literal notranslate"><span class="pre">start()</span></code></a> (it
will run the server forever) or use invoking <a class="reference internal" href="#cheroot.server.HTTPServer.prepare" title="cheroot.server.HTTPServer.prepare"><code class="xref py py-func docutils literal notranslate"><span class="pre">prepare()</span></code></a> and <a class="reference internal" href="#cheroot.server.HTTPServer.serve" title="cheroot.server.HTTPServer.serve"><code class="xref py py-func docutils literal notranslate"><span class="pre">serve()</span></code></a> like this:</p>
```

where the reference is

```
<dl class="method">
<dt id="cheroot.server.HTTPServer.start">
<code class="descname">start</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="headerlink" href="#cheroot.server.HTTPServer.start" title="Permalink to this definition">¶</a></dt>
<dd><p>Run the server forever.</p>
<p>It is shortcut for invoking <a class="reference internal" href="#cheroot.server.HTTPServer.prepare" title="cheroot.server.HTTPServer.prepare"><code class="xref py py-func docutils literal notranslate"><span class="pre">prepare()</span></code></a> then <a class="reference internal" href="#cheroot.server.HTTPServer.serve" title="cheroot.server.HTTPServer.serve"><code class="xref py py-func docutils literal notranslate"><span class="pre">serve()</span></code></a>.</p>
</dd></dl>
``` 

It looks correct for me.

may be I should use some other command for rendering? could you write it to me please?

Reply:
No, looks fine. It's just that normally you don't need to put braces there.